src/transformers/utils/hub.py

```diff
@@ -145,6 +145,24 @@ def is_offline_mode():
 HUGGINGFACE_CO_PREFIX = HUGGINGFACE_CO_RESOLVE_ENDPOINT + "/{model_id}/resolve/{revision}/{filename}"
 HUGGINGFACE_CO_EXAMPLES_TELEMETRY = HUGGINGFACE_CO_RESOLVE_ENDPOINT + "/api/telemetry/examples"
 
+_NO_RETURN = object()
+
+
+def _return_cache_or_none_for_condition(
+    path_or_repo_id: str,
+    full_filename: str,
+    cache_dir: Union[str, Path, None] = None,
+    revision: Optional[str] = None,
+    condition: bool = False,
```

Comment:
I would rather either rename this argument, or have a docstring what the role of `condition` plays.

Reply:
Check the function's name, it's just "condition", if true, this function will return `None`, so I don't know what else the name it can be changed to.
However, I do agree that we need to have some docstrings here for this function.

Reply:
it's just a nit, but I personally would prefer `cond_to_return_none`